regform.html

```diff
@@ -0,0 +1,172 @@
+<!doctype html>
+<html>
+<head>
+    <title> Registration Form</title>
+    <style>
+         #basicuserinfo {
+          background-color: white;
+          border: 3px solid rgb(0, 183, 255);
+          border-radius: 5px;
+          border-bottom: 1px solid rgb(0, 183, 255);
+          padding: 5px;
+          width: 50%;
+        }
+        #addressinfo {
```

Comment:
There was no need to add all these ids for styling only.
Since all the elements you're styling are "fieldsets" 
So you could easily write these styles for all fieldsets. 
And use id only in case you want to modify something in a specific fieldset

Reply:
True that. At the beginning I applied the same style to all fieldsets at once but then I noticed that in the screenshot you sent the fieldsets' borders are different so I thought that the only way to do it is by styling each fieldset separately.

Reply:
nope, they are all the same :)